bundles/org.eclipse.equinox.console/META-INF/MANIFEST.MF

```diff
@@ -8,6 +8,7 @@ Bundle-Vendor: %bundleVendor
 Bundle-Localization: plugin
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Import-Package: org.apache.felix.service.command;version="[1.0,2.0)",
+ org.eclipse.osgi.container;version="[1.6.0,2.0.0]",
```

Comment:
Missed updating the range here for the new API in package version 1.7.

Reply:
Done